WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -1,34 +1,91 @@
-<h1><%= @competition.name %> is confirmed</h1>
+Competition Announcement Team, 
+<br>
+<br>
+<strong><%= @confirmer.name %> has confirmed <%= link_to @competition.name, competition_url(@competition) %>.</strong>
+<br>
+
+<span class="<%= 'alert' if (@competition.start_date - Time.now.to_date).to_i <= 30 %>">
+  The competition will take place on <%= wca_date_range(@competition.start_date, @competition.end_date) %> in <%= (@competition.start_date - Time.now.to_date).to_i %> days.
+  <% if (@competition.start_date - Time.now.to_date).to_i <= 30 %>
+    There are less than 48 hours remaining for the competition to be announced. @<%= "Delegate".pluralize(@competition.delegates.count) %>: please respond to any WCAT concerns as soon as possible to ensure that the competition becomes announced. If the competition becomes less than 28 days away, the WCAT cannot announce it.
+  <% end %>
+  <br>
+  <br>
+</span>
 
 <% if @competition.championships.present? %>
-  <h2 class="alert">This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %></h2>
+  <span class="alert">
+    This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %>.
+    <br>
+    <br>
```

Comment:
There are a lot of these double `<br>` tags in the email template.
Have you considered using `p` (which introduces a bit more spacing) or adding an inline css to make all `span`/`p` have a little more margin bottom?

You also seem to wrap whole paragraph within `span`s, whereas semantically it's supposed to be used to group inline elements together.
Alternatively `p` is the element which defines a paragraph.
While the visual result may be similar, the meaning attached to both constructs is different.

Reply:
That makes a lot of sense! I'll change this over once the other comment I left has an answer on what it should be changed :D

Reply:
Just pushed a change for this. Let me know if that looks better. I still used <br> in the one paragraph with many lines, but I think based on what you wrote that should be fine. Just let me know!

Reply:
Looks great thanks!